types/Product.ts

```diff
@@ -32,7 +32,7 @@ export default interface Product {
   consumerInformationText: string
   fromSweden: boolean
   promotionPrice: Price
-  potentialPromotions: Array<Promotion>
+  potentialPromotions?: Array<Promotion>
```

Comment:
It looks like it will always be in the response, like this if empty:
`"potentialPromotions" : [ ],`
So it will not always as a `interface Promotion`... how do we specify a case like that?

Reply:
@coopjakob thanks, I've changed it!